src/Parser/B3dmParser.js

```diff
@@ -185,66 +150,56 @@ export default {
             const gltfBuffer = buffer.slice(posGltf);
             const headerView = new DataView(gltfBuffer, 0, 20);
 
-            promises.push(new Promise((resolve/* , reject */) => {
-                const onload = (gltf) => {
-                    for (const scene of gltf.scenes) {
-                        scene.traverse(filterUnsupportedSemantics);
-                    }
-                    // Rotation managed
-                    if (gltfUpAxis === undefined || gltfUpAxis === 'Y') {
-                        gltf.scene.applyMatrix4(matrixChangeUpVectorZtoY);
-                    } else if (gltfUpAxis === 'X') {
-                        gltf.scene.applyMatrix4(matrixChangeUpVectorZtoX);
+            const init_mesh = function f_init(mesh) {
+                mesh.frustumCulled = frustumCulled;
```

Comment:
It was set to false initially, any reason for that change?

Reply:
set it back to false, it was a mistake